src/domain/core/src/services/export_service.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::path::PathBuf;
+
+use internal_error::InternalError;
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Service
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#[async_trait::async_trait]
+pub trait ExportService: Send + Sync {
+    async fn export_to_fs(
+        &self,
+        sql_query: &String,
```

Comment:
Would it be better to decouple ExportService and QueryService by simply accepting a `DataFrame` here instead of SQL?

Reply:
we need to create specific session context with single target partition and specific rows count per target file.

Reply:
Makes sense! I will change it.

Reply:
@sergiimk Can you explain how `DataFrame` object will help. Isn't this `DataFusion`'s specific trait?
I understand Spark and Flink have a similar concept, but I guess it's not the same interface?
Or do we plan to wrap muiltiple types with our own abstraction?

Reply:
`DataFrame` can wrap Arrow buffers and our long-term goal is to make Arrow the only data interchange format in Kamu. 

There is a high chance Spark and other engines will soon natively support Arrow format. If they don't we'll imlement adapters. But the idea is that `DataFrame` will be able to represent output data from any engine in ODF, not only DataFusion.